src/analysis/seminorm.lean

```diff
@@ -75,41 +75,10 @@ variables (𝕜) [has_scalar 𝕜 E]
 `A` by elements of sufficiently large norms. -/
 def absorbs (A B : set E) := ∃ r, 0 < r ∧ ∀ a : 𝕜, r ≤ ∥a∥ → B ⊆ a • A
 
-/-- A set is absorbent if it absorbs every singleton. -/
-def absorbent (A : set E) := ∀ x, ∃ r, 0 < r ∧ ∀ a : 𝕜, r ≤ ∥a∥ → x ∈ a • A
```

Comment:
Why is this (and all the other stuff in `analysis/seminorm`) being moved? I'm not saying it shouldn't be, just asking for explanation.

Reply:
the diff looks weird, essentially I moved stuff up so that there are less assumptions on the lemmas. Somehow no linter caught that.

Reply:
I moved this part to a separate PR.